cms/djangoapps/contentstore/views/course.py

```diff
@@ -528,6 +528,9 @@ def course_index(request, course_key):
     # A unit may not have a draft version, but one of its components could, and hence the unit itself has changes.
     with modulestore().bulk_operations(course_key):
         course_module = get_course_and_check_access(course_key, request.user, depth=None)
+        if not course_module:
+            log.error(u"Unable to find course with course key %s", course_key)
```

Comment:
I like the thought! But I think logging an error here will just add noise to the logs. Not necessary.